src/components/answer/InputAnswer.jsx

```diff
@@ -6,7 +6,7 @@ import FormUtils from "../../util/FormUtils";
 import YASQE from "yasgui-yasqe";
 import YATE from "yasgui-yasqe";
 import { ConfigurationContext } from "../../contexts/ConfigurationContext";
-import "@triply/yasgui/build/yasgui.min.css";
```

Comment:
@LaChope I am not sure why we do that ... so what are consequences of this decision?

I assume this:
- we did not depend on styling of yasgui component and thus whenever we upgrade new version of yasgui, we need to remember to also generate yasgui.min.css as well 
- so what did we get in return? did we need to modify yasgui ? (If so why do we add minified version and not well formated version?)
-